config/locales/en.yml

```diff
@@ -189,11 +189,12 @@ en:
         supporting_documents_html:
           "Would you like to attach supporting documents? (<span class='text-red'>Required</span>)"
       application_details_form:
+        application_link: Link to online application
         contact_email_html: "Contact email (<span class='text-red'>Required</span>)"
-        school_visits: School visits
+        contact_number: Contact phone number
```

Comment:
It's missing on the Figma, but this is a required field, so `contact_number_html: "Contact phone number (<span class='text-red'>Required</span>)"`